devtools.js

```diff
@@ -79,13 +82,22 @@ chrome.devtools.panels.elements.createSidebarPane(
             let comps = {};
             
             // adds computed properties to data properties and returns the result
-            let computeds = Object.keys(comp)
-                .filter(key => !key.startsWith('${'))
-                .reduce((acc, key) => {
-                    acc[key] = comp[key].getter();
-                    return acc;
-                }, comps);
-            
+            if (majorVersion === "0.9") {
```

Comment:
A switch statement might be cleaner here?

Reply:
Currently just a single thing to check, if we need more cases with later versions then change to a switch at that point imo.